src/middlewares/login.Vendedores.js

```diff
@@ -0,0 +1,30 @@
+const bcrypt = require("bcryptjs");
+
+const LoginVendedor = async (req, res) => {
+  const { clave, password } = req.body;
+
+  try {
+    const Vendedor = await Vendedor.findOne({ clave });
+    if (!Vendedor) {
+      return res.status(400).json({ message: "vendedor no encontrado" });
+    }
+
+    const isPasswordValid = await bcrypt.compare(password, Vendedor.password);
+    if (!isPasswordValid) {
+      return res.status(400).json({ message: "contraseña incorrecta" });
+    }
+
+    const token = jwt.sign({ vendedorId: Vendedor._id }, JWT_SECRET, {
+      expiresIn: "5h",
+    });
+
+    res.cookie("sesion", token, { maxAge: 900000, httpOnly: true });
```

Comment:
vos colocaste que el token tenia un tiempo de 5h, por lo tanto el maxAge aqui tambien deberia ser de 5h . 
```suggestion
    res.cookie("sesion", token, { maxAge: 18000000, httpOnly: true });
```